presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -361,6 +370,257 @@ public void testDecimalExceedingPrecisionMaxConvertedToVarchar()
                 "'12345678901234567890123456789012345678901234567890'");
     }
 
+    @Test
+    public void testDecimalExceedingPrecisionMaxWithExceedingIntegerValues()
+    {
+        JdbcSqlExecutor jdbcSqlExecutor = new JdbcSqlExecutor(postgreSqlServer.getJdbcUrl());
+
+        try (TestTable testTable = new TestTable(
+                jdbcSqlExecutor,
+                "tpch.test_exceeding_max_decimal",
+                "(d_col decimal(65,25))",
+                asList("1234567890123456789012345678901234567890.123456789", "-1234567890123456789012345678901234567890.123456789"))) {
+            String testTableName = testTable.getName().split("[.]")[1];
```

Comment:
[optional] Maybe `TestTable` constructor could accept `SchemaTableName` as second argument and here you could call `getTableName()` without using `split` and regex matching.

Reply:
Adding a `schemaTableName` to `TestTable` is  a good option, but since there are places that using it without a schema name, I will go now with a quick change that skips the usage of `split` and regexp matching